Recrutify/Recrutify.DataAccess/Repositories/UserRepository.cs

```diff
@@ -15,6 +17,14 @@ public UserRepository(IOptions<MongoSettings> options)
         {
         }
 
+        public async Task<Dictionary<Guid, string>> GeStaff(IEnumerable<Guid> ids)
+        {
+            var filter = _filterBuilder.In(u => u.Id, ids);
+            var users = await GetCollection().Find(filter).ToListAsync();
+            var userDictionary = users.ToDictionary(k => k.Id, v => $" {v.Name} {v.Surname}");
+            return userDictionary;
```

Comment:
You can not create var, just write return on prev line

Reply:
done